### PR TITLE
hover underlines on links in tracker proofs. moved flow types

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: react-native/react/tracker/proofs.render.desktop.js [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 import React, {Component} from '../base-react'
 import commonStyles, {colors} from '../styles/common'
-
 import {checking, revoked} from '../constants/tracker'
 import {metaNew, metaUpgraded} from '../constants/tracker'
+import {remote} from 'electron'
```
Comment (MarcoPolo) [inline react-native/react/tracker/proofs.render.desktop.js]: :sheep: can't you do `{remote: {shell}} from 'electron'`

**Comment by MarcoPolo** [comment]
:+1: Awesome thank you for moving the types :)

**Comment by MarcoPolo** [comment]
one thing before merging, do a search for where `*.render.types` are being used. I think it's being used in reducers/tracker.js as well

File: react-native/react/tracker/proofs.render.desktop.js (hunk 1 above)
Comment (chrisnojima) [inline react-native/react/tracker/proofs.render.desktop.js]: i had problems with that not working before with babel, maybe they fixed it... also the other branch changed this code so it won't really work anymore. it's now
```
const shell = electron.shell || electron.remote.shell
```

i really REALLY hate that electron exposes different apis like this

**Comment by chrisnojima** [comment]
i'll merge this in the morning. i think we should start using a git precommit hook to ensure our flow stuff isn't broken cause i do that too much

**Comment by cjb** [comment]
LGTM!  Options other than a precommit hook for flow:

* get the CI to run it
* get a bot to comment on each PR with any flow issues

**Comment by chrisnojima** [comment]
I vote for all 3 :)

**Pull request merged**